nixos/modules/services/network-filesystems/seaweedfs.nix

```diff
@@ -0,0 +1,597 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.seaweedfs;
+  baseDir = "/var/lib/seaweedfs";
+in
+{
+  options.services.seaweedfs = {
+    package = lib.mkOption {
+      type = lib.types.package;
+      default = pkgs.seaweedfs;
+      defaultText = lib.literalExpression "pkgs.seaweedfs";
+      description = "The SeaweedFS package to use.";
+    };
+
+    master = {
+      enable = lib.mkEnableOption "SeaweedFS master server";
+
+      port = lib.mkOption {
+        type = lib.types.port;
+        default = 9333;
+        description = "Port for master server.";
+      };
+
+      grpcPort = lib.mkOption {
+        type = lib.types.port;
+        default = 19333;
+        description = "gRPC port for master server.";
+      };
+
+      ip = lib.mkOption {
+        type = lib.types.str;
+        description = "IP address to bind to.";
+      };
+
+      ipBind = lib.mkOption {
+        type = lib.types.str;
+        default = "";
+        description = "IP address to bind to. If empty, defaults to same as ip.";
+      };
```

Comment:
I don't understand the difference between those

Reply:
Is not the same.
That is strange but is options implemented by seaweeds.